Pod/Classes/URBNAlertView.m

```diff
@@ -39,6 +39,7 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
             self.customView = dummyView;
         }
         else {
+            customView.translatesAutoresizingMaskIntoConstraints = NO;
```

Comment:
You could put `self.customView.translatesAutoresizingMaskIntoConstraints = NO;` outside this if-else to reduce redundancy and prevent a case where one is set and the other isn’t.